project/api/models/QuizSession.py

```diff
@@ -1,16 +1,18 @@
 from __future__ import unicode_literals
 
+from django.utils import timezone
+
 from django.db import models
 
-from . import Quiz, QuizResult, User
+from . import Quiz, User
 
 class QuizSession(models.Model):
     owner = models.ForeignKey(User, on_delete=models.CASCADE)
     quiz = models.ForeignKey(Quiz, on_delete=models.CASCADE)
-    quiz_result = models.ForeignKey(QuizResult, on_delete=models.CASCADE)
     is_locked = models.BooleanField(default=False)
-
-    # NOTE: It is possible to have a quiz result that does not match the quiz
```

Comment:
This doesn't really matter - but we should remember that it is still possible to have responses on a QuizSession that do not match the quiz of the QuizSession

Reply:
make a card detailing this and we can block that from happening at the api layer.  We'll pull the quiz from the question from the answer in the response and compare it to the quiz associated with the session

Reply:
Done